BrowserKit/README.md

```diff
@@ -0,0 +1,3 @@
+# BrowserKit
+
+A collection of projects that are useful for building a browser
```

Comment:
We could add steps here on how to run this project and it's tests

Reply:
You don't need to do anything to run it, it works like any other SPM package, you just import what you want. I can add a comment about running the tests tho.